notebooks/a01_manifold_class.ipynb

```diff
@@ -0,0 +1,1630 @@
+{
```

Comment:
<p>Maybe add something like: "Observe the abstract methods denoted with "@abc.abstractmethod" that do not contain any implementation, but serve as template for the subclasses to overwrite. The code of manifold can also be found here: https://github.com/geomstats/geomstats/blob/master/geomstats/geometry/manifold.py"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1340/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>done</p>